File: CS/Form1.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;

namespace WindowsFormsApplication4 {
    public partial class Form1 : DevExpress.XtraEditors.XtraForm {
        public Form1() {
            InitializeComponent();
        }

        private void btnShowRadialMenu_ItemClick(object sender, DevExpress.XtraBars.ItemClickEventArgs e) {
            Point pt = GetMenuLocation();
            myRadialMenu1.ShowPopup(pt);
        }
        private Point GetMenuLocation() {
            Point pt = new Point();
            pt.X = Location.X + Size.Width / 2;
            pt.Y = Location.Y + Size.Height / 2;
            return pt;
        }
    }
}

File: VB/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.Drawing
Imports System.Linq

Namespace WindowsFormsApplication4
    Partial Public Class Form1
        Inherits DevExpress.XtraEditors.XtraForm

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub btnShowRadialMenu_ItemClick(ByVal sender As Object, ByVal e As DevExpress.XtraBars.ItemClickEventArgs) Handles btnShowRadialMenu.ItemClick
            Dim pt As Point = GetMenuLocation()
            myRadialMenu1.ShowPopup(pt)
        End Sub
        Private Function GetMenuLocation() As Point
            Dim pt As New Point()
            pt.X = Location.X + Size.Width \ 2
            pt.Y = Location.Y + Size.Height \ 2
            Return pt
        End Function
    End Class
End Namespace
